pkgs/top-level/python-packages.nix

```diff
@@ -9444,6 +9444,13 @@ in {
     phantomjsSupport = false;
   };
 
+  yt-dlp = callPackage ../tools/misc/yt-dlp { };
+
+  yt-dlp-light = callPackage ../tools/misc/yt-dlp {
+    ffmpegSupport = false;
+    phantomjsSupport = false;
+  };
```

Comment:
Can packages use this as a lib? if not we don't need this entry at all.

Reply:
It's mostly similar to youtube-dl, so it's still a lib